OSDOCS-999 Bring forward S2I content (#27435)
openshift/openshift-docs: openshift_images/using_images/customizing-s2i-images.adoc

[id="customizing-s2i-images"]
= Customizing source-to-image images
include::modules/common-attributes.adoc[]
:context: customizing-s2i-images

> Source-to-image (S2I) builder images include assemble and run scripts, but the default behavior of those scripts is not suitable for all users. This topic covers a few approaches for customizing the behavior of an S2I builder that includes default scripts.

Reviewer: I only see one method, and the style guide says to avoid using self-referential language. Maybe:

```suggestion
Source-to-image (S2I) builder images include assemble and run scripts, but the default behavior of those scripts is not suitable for all users. You can customize the behavior of an S2I builder that includes default scripts.
```

> include::modules/images-using-invoking-scripts-embedded-s2i.adoc[leveloffset=+1]

Reviewer: I suggest using images-using-customizing-s2i-images-scripts-embedded.adoc instead.
